org.techxartisan.openterfaceQT/org.techxartisan.openterfaceQT.yaml

```diff
@@ -0,0 +1,54 @@
+app-id: org.techxartisan.openterfaceQT
+runtime: org.kde.Platform
+runtime-version: "6.7"
+sdk: org.kde.Sdk
+command: openterfaceQT
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --device=all
+  - --filesystem=/run/udev:ro
+
+modules:
+  - name: libusb
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://github.com/libusb/libusb/releases/download/v1.0.26/libusb-1.0.26.tar.bz2
+        sha256: 12ce7a61fc9854d1d2a1ffe095f7b5fac19ddba095c259e6067a46500381b5a5
+    build-commands:
+      - ./configure --prefix=/app
+      - make
+      - make install 
+
+  - name: openterfaceQT
+    buildsystem: simple
+    sources:
+      - type: git
+        url: https://github.com/TechxArtisanStudio/Openterface_QT.git
+        branch: main
+
+    config-opts: 
+      - "CONFIG += no_install"
+      - "LIBS += -L/app/lib/ -lusb-1.0"
+      - "INCLUDEPATH += /app/include/libusb-1.0"
+      
+    build-options:
+      env:
+        PKG_CONFIG_PATH: /app/lib/pkgconfig:${PKG_CONFIG_PATH}
+        LIBRARY_PATH: /app/lib:${LIBRARY_PATH}
+        LD_LIBRARY_PATH: /app/lib:${LD_LIBRARY_PATH}
+        QT_SELECT: qt6
+    build-commands:
+      - lupdate openterfaceQT.pro
+      - lrelease openterfaceQT.pro
+      - qmake "LIBS+=-L/app/lib -lusb-1.0" "INCLUDEPATH+=/app/include/libusb-1.0"
+      - make -j$(nproc)
```

Comment:
this should never be `nproc` but instead be `$FLATPAK_BUILDER_N_JOBS`

But this look like it could be `buildsystem: qmake`